frontend/app/src/components/taxreport/TaxReportEvents.vue

```diff
@@ -18,27 +18,63 @@
               Taxable Received in {{ currency }}
             </template>
             <template #item.paidInProfitCurrency="{ item }">
-              {{ item.paidInProfitCurrency | formatPrice(floatingPrecision) }}
+              {{
+                item.paidInProfitCurrency
+                  | formatPrice(
+                    amountDisplayFormat,
+                    floatingPrecision,
+                    currency.unicode_symbol
+                  )
+              }}
```

Comment:
Some DRY code here that we can clean by putting that on a component method?

Reply:
It seems that we missed that when we introduced the `AmountDisplay.vue`, we can create a followup issue for replacing that with the component.

Reply:
Why don't we just introduce using AmountDisplay for these components with this PR? It would amount to the same thing.

Reply:
If CPatchane wants to do that as part of his PR sure. But since this was not in the original scope of the issue, we can still merge and follow up with a new issue.

Reply:
He's already making changing to TaxReportEvents so that it works with this PR (which is outside the scope of this issue), it doesn't make sense to make changes that will just immediately be overwritten with another issue.

Reply:
I changed this component to use the `AmountDisplay.vue` component here